android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/WorkerBinder.kt

```diff
@@ -155,10 +294,42 @@ object WorkerBinder {
     worker: Worker
   ) {
     workerLifecycle.subscribe { workerEvent: WorkerEvent ->
+
       when (workerEvent) {
-        WorkerEvent.START -> worker.onStart(WorkerScopeProvider(workerLifecycle.hide()))
-        else -> worker.onStop()
+        WorkerEvent.START -> {
+          val onStartTotalDurationMilli = measureTimeMillis {
+            worker.onStart(WorkerScopeProvider(workerLifecycle.hide()))
```

Comment:
Adding monitoring now will allow to monitor long Worker.onStart durations happening in main thread in current WorkerBinder.bind usages